src/Storages/ExternalStream/Kafka/WriteBufferFromKafka.cpp

```diff
@@ -0,0 +1,80 @@
+#include "WriteBufferFromKafka.h"
+#include <cmath>
```

Comment:
`cmath` for ?

Reply:
The vector header was added accidentially by the language server.